servicebus/data-plane/azure-servicebus/pom.xml

```diff
@@ -0,0 +1,99 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+	<artifactId>azure-servicebus</artifactId>	
+	<version>2.0.0</version>
+
+  <name>Microsoft Azure Service Bus</name>
+  <description>Java library for Azure Service Bus</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <parent>
+		<groupId>com.microsoft.azure</groupId>
+		<artifactId>azure-servicebus-parent</artifactId>
+		<version>2.0.0</version><!-- Need not change for every release -->
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+	</properties>
+
+	<build>
+		<resources>
+			<resource>
+				<directory>resources</directory>
+				<filtering>true</filtering>
+			</resource>
+		</resources>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+					<optimize>true</optimize>
+					<showDeprecation>true</showDeprecation>
+					<showWarnings>true</showWarnings>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <forkCount>0</forkCount> <!-- Speeds up test execution using multiple JVMs. Set it to 0 if you want to run test sequentially in the build vm itself -->
+          <reuseForks>true</reuseForks>
+        </configuration>
+      </plugin>
+		</plugins>
+	</build>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.qpid</groupId>
+      <artifactId>proton-j</artifactId>
+    </dependency>
+      <dependency>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>qpid-proton-j-extensions</artifactId>
+      </dependency>
+    <dependency>
+       <groupId>com.microsoft.azure</groupId>
+       <artifactId>adal4j</artifactId>
+       <version>1.3.0</version>
```

Comment:
version 1.3.0 can't be removed because existing default version produced following errors:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project azure-servicebus: Compilation failure: Compilation failure:
[ERROR] /C:/Users/shafang/work/azure-sdk-for-java/servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/security/AzureActiveDirectoryTokenProvider.java:[104,20] com.microsoft.azure.servicebus.security.AzureActiveDirectoryTokenProvider.FutureCompletingAuthenticationCallback is not abstract and does not override abstract method onSuccess(java.lang.Object) 
in com.microsoft.aad.adal4j.AuthenticationCallback
[ERROR] /C:/Users/shafang/work/azure-sdk-for-java/servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/security/AzureActiveDirectoryTokenProvider.java:[121,9] 
method does not override or implement a method from a supertype

Reply:
It looks like you need to implement the onSuccess method. I would prefer that you do this rather than bring in another adal4j version needlessly. Please investigate more deeply. Thanks!